src/sentry/api/fields/actor.py

```diff
@@ -9,6 +9,13 @@ class Actor(namedtuple("Actor", "id type")):
     def get_actor_id(self):
         return "%s:%d" % (self.type.__name__.lower(), self.id)
 
+    def get_type_string(self):
+        return self.type.__name__.lower()
+
+    @classmethod
+    def from_model(cls, actor):
+        return actor_type_to_model(actor.type).objects.get(actor_id=actor.id)
```

Comment:
Does this belong here? It doesn't return an actual `Actor`, it returns the model instead. Tbh, I haven't thought a lot about how we'll have this class and the model work together.

Reply:
Yeah me either but I've been trying to figure about how to unify them better as I work with it

Reply:
We definitely shouldn't return a different class here though, confusing because we already have `from_*` methods that return the non-model `Actor`